bin/build-helper

```diff
@@ -5,8 +5,11 @@ set -u
 set -o pipefail
 
 BASE_DIR=$(dirname $0)/..
-NODE_ENV=${NODE_ENV:-""}
-REGISTRY=${REGISTRY:=""}
+: ${NODE_ENV:=""}
+: ${REGISTRY:=""}
+if [[ -n "$REGISTRY" ]]; then  # if non-empty
+  REGISTRY="$(echo $REGISTRY | sed 's#/$##')/"  # ensure '/' suffix
```

Comment:
I have verified this like (which is the fix) on the terminal and it work as expected, proactively remove the trailing slash (if exist) so it can be explicitly added